README.md

```diff
@@ -2,46 +2,51 @@
 
 ![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780) 
 
-## Setup
-* Clone the repo
-* Install dependencies `pip install -r requirements.txt`  
-
-## Set BrowserStack Credentials 
-* Export the BrowserStack username and access key as environment variables
+## Prerequisite
+```
+python3 should be installed
+```
 
-#### For Linux/MacOS
-  ```
-  export BROWSERSTACK_USERNAME=<browserstack-username>
-  export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+## Setup
+* Clone the repo 
   ```
-#### For Windows
+  git clone -b sdk https://github.com/browserstack/robot-browserstack.git
   ```
-  setx BROWSERSTACK_USERNAME=<browserstack-username>
-  setx BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+* Install dependencies 
   ```
+  pip install -r requirements.txt
+  ```  
+
+## Set BrowserStack Credentials
+* Add your BrowserStack username and access key in the `browserstack.yml` config fle.
+* You can also export them as environment variables, `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY`:
+
+  #### For Linux/MacOS
+    ```
+    export BROWSERSTACK_USERNAME=<browserstack-username>
+    export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+    ```
+  #### For Windows
+    ```
+    setx BROWSERSTACK_USERNAME=<browserstack-username>
```

Comment:
Add set command too, setx sets the env vars in the next command prompt.

Reply:
[addressed](https://github.com/browserstack/robot-browserstack/pull/4/commits/61aa0ac0bbb9ae27cc406b45f9532e4854816f10)